src/main/java/ca/gc/cyber/ops/assemblyline/java/client/model/submission/Submission.java

```diff
@@ -50,6 +50,10 @@
      * List of result keys (SHA256.ServiceName.ServiceVersion.Config)
      */
     List<String> results;
+    /**
+     * The filescore key, used in deduplication. This is a non-unique key, that is shared by submissions that may be processed as duplicates.
+     */
+    String scanKey;
```

Comment:
I guess filescore key = scan key (I don't know anything about those)? Just making sure, since the doc and field name differs.

Reply:
I hadn't noticed that, but I just copied the comment from the python model: https://github.com/CybercentreCanada/assemblyline-base/blob/76d80e0c86303375f8a10a8fca02529b786ddae2/assemblyline/odm/models/submission.py#L127